controllers/app.controller.js

```diff
@@ -12,6 +12,20 @@ exports.getTopic = (req, res, next) => {
     });
 };
 
+exports.getArticleByID = (req, res, next) => {
+  const validArticleId = !isNaN(parseInt(req.params.article_id));
+  if (validArticleId) {
+    articleById(req.params.article_id).then((result) => {
+      if (result === 0) {
+        res.status(404).send({ msg: "Article not found" });
+      }
+      res.status(200).send(result);
```

Comment:
please send as a properties on the body with a descriptive key.   ie articles instead of result and .send({articles})